shared/multiserieshorizontalbarchart.js

```diff
@@ -249,7 +273,7 @@ function renderGroupedBarChart(elem, data, width, height, keys, options) {
     .attr("fill", "#000")
     .attr("font-weight", "bold")
     .attr("text-anchor", "start")
-    .text(barOptions.yAxisTitle);
+  //   .text(barOptions.yAxisTitle);
```

Comment:
```suggestion
```

since yAxisTitle is added above, we can delete this line (and the `barOptions` object on line 190)